bandwidth/bandwidth-impl/src/main/java/com/duckduckgo/bandwidth/impl/BandwidthPixelName.kt

```diff
@@ -19,11 +19,21 @@ package com.duckduckgo.bandwidth.impl
 import com.duckduckgo.app.statistics.pixels.Pixel
 
 enum class BandwidthPixelName(override val pixelName: String) : Pixel.PixelName {
-    BANDWIDTH("m_bandwidth")
+    BANDWIDTH("m_bandwidth"),
```

Comment:
Should we also remove the `RealBandwidthCollector` as part of this PR ?

Reply:
I'll remove it in a follow-up PR 👍